src/renderer/helpers/utils.js

```diff
@@ -634,7 +635,10 @@ export function toDistractionFreeTitle(title, minUpperCase = 3) {
   }
 
   const reg = RegExp(`[\\p{Lu}|']{${minUpperCase},}`, 'ug')
-  return title.replace(reg, x => capitalizedWord(x.toLowerCase()))
+  return title
+    .replaceAll(/!{2,}/g, '!')
+    .replaceAll(/[!?|]{2,}/g, '?')
```

Comment:
Why `[!?|]{2,}` not `\?{2,}`

Reply:
To deal with strings like `TITLE ???!??!!!!!?`

Reply:
1. `/[!?]{2,}/g` is enough (without `|`)
2. Please provide test case (video) for title with `?!?!?`

Reply:
Here is an example 

https://www.youtube.com/watch?v=-SZwcg73BYs

Here is what is looks like filtered 

![image](https://github.com/FreeTubeApp/FreeTube/assets/30714020/371791b5-e75f-445e-80d7-e6301f86a177)